commcare_connect/opportunity/models.py

```
@@ -993,6 +997,26 @@ class Meta:
+    blob_id = models.CharField(max_length=255, default=uuid4, unique=True)
+    name = models.CharField(max_length=255)
+    content_type = models.CharField(max_length=255, null=True)
+    content_length = models.IntegerField()
+    transcript = models.TextField(blank=True, default="")
+    translation = models.TextField(blank=True, default="")
+    date_created = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = [
+            ("user_visit", "name"),
+        ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)